qcodes_contrib_drivers/drivers/Keysight/Keysight_E8267D.py

```diff
@@ -109,10 +115,16 @@ def __init__(self, name, address, step_attenuator=False, **kwargs):
                            set_cmd='OUTP {}',
                            get_parser=parse_on_off,
                            vals=on_off_validator)
+        self.add_parameter(name='output_rf',
+                           source=self.status,
+                           parameter_class=DelegateParameter)
         self.add_parameter(name='modulation_rf_enabled',
                            get_cmd='OUTP:MOD?',
                            set_cmd='OUTP:MOD {}',
                            val_mapping=on_off_mapping)
+        self.add_parameter(name='modulation_rf',
+                           source=self.modulation_rf_enabled,
+                           parameter_class=DelegateParameter)
```

Comment:
each of these parameters worth a comment in the code explaining "why"

Reply:
If the aim is to make the parameters here consistent with the existing parameters in the other driver one should also consider doing this the other way around and leaving the existing parameters in this driver as deprecated delegates

Reply:
now, issuing deprecation warning which explains they reason why